library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapter.java

```diff
@@ -51,7 +56,7 @@ public V instantiateItem(ViewGroup container, int position) {
      * @param position  the position of the data set that is to be represented by this view
      * @param viewState the state of the view
      */
-    protected void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
+    private void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
```

Comment:
It is used on one of the project for sure, we need to know when the views are bound so we can run some logic in the respective views.